packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.html

```diff
@@ -102,57 +106,121 @@ <h3 data-translate="telecom_pack_migration_doing"></h3>
+                                ></td>
+                                <td
+                                    class="text-right"
+                                    data-title="{{ 'telecom_pack_migration_confirm_header_quantity' | translate }}"
+                                    data-ng-bind="$ctrl.QUANTITY"
+                                ></td>
+                                <td
+                                    class="text-price text-right"
+                                    data-ng-bind="$ctrl.process.selectedOffer.providerAI.text"
                                     data-title="{{ 'telecom_pack_migration_confirm_header_total' | translate }}"
                                 ></td>
                             </tr>
                             <tr
-                                data-ng-if="$ctrl.process.selectedOffer.options.gtr_ovh && $ctrl.process.selectedOffer.options.gtr_ovh.selected"
+                                data-ng-if="$ctrl.process.selectedOffer.gtrComfortActivated"
                             >
                                 <th
                                     scope="row"
-                                    data-translate="telecom_pack_migration_confirm_resume_gtr_ovh"
+                                    data-translate="telecom_pack_migration_confirm_comfort_activation"
                                     data-title="{{ 'telecom_pack_migration_confirm_header_entitled' | translate }}"
                                 ></th>
                                 <td
                                     class="text-price text-right"
-                                    data-ng-bind="$ctrl.process.selectedOffer.options.gtr_ovh.optionalPrice.text"
+                                    data-ng-bind="$ctrl.process.selectedOffer.gtrComfortFees.text"
                                     data-title="{{ 'telecom_pack_migration_confirm_header_price' | translate }}"
                                 ></td>
                                 <td
                                     class="text-right"
                                     data-title="{{ 'telecom_pack_migration_confirm_header_quantity' | translate }}"
-                                >1</td>
+                                    data-ng-bind="$ctrl.QUANTITY"
+                                ></td>
                                 <td
                                     class="text-price text-right"
-                                    data-ng-bind="$ctrl.process.selectedOffer.options.gtr_ovh.optionalPrice.text"
+                                    data-ng-bind="$ctrl.process.selectedOffer.gtrComfortFees.text"
+                                    data-title="{{ 'telecom_pack_migration_confirm_header_total' | translate }}"
+                                ></td>
+                            </tr>
+                            <tr
+                                data-ng-repeat="option in $ctrl.choosedAdditionalOptions track by $index"
```

Comment:
No `id` on option to use for the track by?

Reply:
no need of ```id```